main.py

```diff
@@ -57,7 +57,9 @@
 @app.on_event("startup")
 async def startup():
     """Connect to database on startup."""
-    asyncio.create_task(bot.start(config["DISCORD_BOT_TOKEN"]))
+    if "DISCORD_BOT_TOKEN" in config and config["DISCORD_BOT_TOKEN"]:
```

Comment:
```suggestion
    if "DISCORD_BOT_TOKEN" in config:
```

Don't need to check this twice.